### DISCUSSION TITLE
Add collections to be able to group saved snippets

### DISCUSSION THREAD

**Discussion by kievsash** [dawnlabs/carbon#1559] [Ideas]
### Expected Behavior

- [x] 1. Saved snippets can be grouped to collections.
- [x] 2. Copy URL for specific snippet possibility so one can specify it in an article.
- [x] 3. Named snippets - one can specify name for the specific snippet
- [x] 4. Search snippet by name

### Actual Behavior

Saving snippet as one-level big collection with no grouping and search.

Thank you in advance!

**Comment by mfix22** [comment]
These are all _great_ suggestions! Naming snippets is something we want to add first, and then search, but grouping them into folders is also a good idea!

**Comment by mfix22** [comment]
@kievsash snippet names is getting released today, partly thanks to you! 🙏 https://github.com/carbon-app/carbon/pull/992

**Comment by kievsash** [comment]
Thank you! Should check it!
